modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -27,12 +27,32 @@ kubectl describe nodes
 [[memory]]
 == Configure memory resources
 
-On a worker node, Kubernetes and Redpanda processes are running at the same time, including the Seastar subsystem that is built into the Redpanda binary. Each of these processes consumes memory. You can configure the memory resources that are allocated to these processes.
+On a worker node, Kubernetes and Redpanda processes are running at the same time, including the Seastar subsystem that is built into the Redpanda binary. Each of these processes consumes memory. Proper configuration of memory resources ensures optimal performance and stability for your Redpanda deployment.
```

Comment:
I don't think it makes sense to talk about Seastar as a separate subsystem. It's a framework that redpanda is built in and doesn't really do anything besides running redpanda. Do we discuss it like this elsewhere in our docs?